pkg/webui/console/containers/gateway-location-form/gateway-location-form.js

```diff
@@ -65,6 +69,13 @@ const validationSchema = Yup.object().shape({
   }),
   location_public: Yup.bool(),
   update_location_from_status: Yup.bool(),
+  _placement: Yup.string()
+    .oneOf(['PLACEMENT_UNKNOWN', 'INDOOR', 'OUTDOOR'])
+    .when('update_location_from_status', {
```

Comment:
This is unrelated; the location can be updated from status messages, and still the placement can be indoor or outdoor. The status message does not include the placement, only the location.

Reply:
According to https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.14/api/gateway.proto#L315 `placement` is a part of gateway antennas list. Should we then register a dummy antenna with specific placement if `update_location_from_status` is selected?

Current implementation strips the `antennas` entry if `update_location_from_status` is selected by the user.

Reply:
Yes indeed, we should. It's not a dummy antenna. Antenna has valuable information like placement and gain: it's not only about location (anymore). We can assume that each gateway has at least 1 antenna. If there is no information about antennas, we allow no antennas to be set.

Reply:
So, if I understood that correctly setting placement should work as follows:
1. The user sets antenna location manually (setting latitude, longitude, altitude), then we simply add the `placement` field within that antenna.
2. The user selects `update_location_from_status`, then location fields are disabled (latitude, longitude, altitude). If there is an antenna, then we allow setting `placement`, otherwise it is disabled.

Is this correct?

Reply:
Yes. The `update_location_from_status` is only for location.

The user can always configure `gain` and `placement` for the antenna.